clang/lib/Driver/Driver.cpp

```diff
@@ -3551,6 +3558,16 @@ class OffloadingActionBuilder final {
           Action *DeviceBECompileAction;
           ActionList BEActionList;
           BEActionList.push_back(SPIRVTranslateAction);
+          for (auto *A : FPGAObjects) {
+            // Send any known objects through the unbundler to grab the
+            // dependency file associated.
+            ActionList AL;
+            AL.push_back(A);
+            Action *UnbundleAction =
+                C.MakeAction<OffloadUnbundlingJobAction>(AL,
```

Comment:
Am I right to assume this would affect the phases list? If so, could the LITs be updated to reflect the additional unbundler phase?

Reply:
Yes, impacts phases.  Added additional test to check the phases when unbundling deps from an object.